imap/command/logout.go

```diff
@@ -0,0 +1,22 @@
+package command
+
+import (
+	"fmt"
+	"github.com/ProtonMail/gluon/imap/parser"
+)
+
+type LogoutCommand struct{}
+
+func (l LogoutCommand) String() string {
+	return fmt.Sprintf("NOOP")
```

Comment:
logout

Reply:
how could it pass the test ?

Reply:
This is just for debug logging, not for the actual parsing.

Fixed.